wpeverest/user-registration
# Feature - Multiple choice Field

### All Submissions:






### Changes proposed in this Pull Request:

Introducing a new payment field called Multiple Choice.

### How to test the changes in this Pull Request:

1. Navigate to form builder and drag multiple choice.
2. Under Field Options of multiple choice field you will find few settings. Change the settings as per your convenience.
3. Navigate to the form in the frontend and verify if multiple choice  is rendered properly and the multiple choice  is working fine.

### Dependent PR
https://github.com/wpeverest/user-registration-payments/pull/12
https://github.com/wpeverest/user-registration-stripe/pull/11
https://github.com/wpeverest/user-registration-conditional-logic/pull/18

### Types of changes:




### Other information:




### Changelog entry
> Feature - Multiple choice Field
